src/Main.hs

```diff
@@ -49,10 +57,13 @@ to_input m
   | takeExtension m `elem` suffixes = File m
   | otherwise                       = Module (splitModName m)
 
+resolveNodes :: Nodes -> [Int] -> [[String]]
+resolveNodes (_,map) xs = fromMaybe (error "Missing nodeId") . flip IMap.lookup map <$> xs
 
+type Nodes'  = Trie.Trie String [((NodeT,String),Int)]
+               -- Maps a path to:   ((node, label), nodeId)
```

Comment:
Instead of renaming `Nodes` to `Nodes'`, maybe we can keep leave it as is, and name the other pair time something different